### PR TITLE
Init boilerplate

### PR DESCRIPTION
**Pull request by fabysshk**
DZ

### CODE REVIEW

File: src/stylesheets/abstracts/_media-queries.scss [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    }
+  
+  // If the breakpoint doesn't exist in the map.
+  } @else {
+
+    // Log a warning.
+    @warn 'Invalid breakpoint: #{$breakpoint}.';
+  }
+}
+
```
Comment (fabysshk) [inline src/stylesheets/abstracts/_media-queries.scss]: $mq-phone: 400px;
$mq-phone-wide: 480px;
$mq-phablet: 560px;
$mq-tablet-small: 640px;
$mq-tablet: 768px;
$mq-tablet-wide: 1024px;
$mq-desktop: 1248px;
$mq-desktop-wide: 1440px;

$breakpoints: (
        "phone":        $mq-phone,
        "phone-wide":   $mq-phone-wide,
        "phablet":      $mq-phablet,
        "tablet-small": $mq-tablet-small,
        "tablet":       $mq-tablet,
        "tablet-wide":  $mq-tablet-wide,
        "desktop":      $mq-desktop,
        "desktop-wide": $mq-desktop-wide
);

@mixin mq($width, $type) {
    @if map_has_key($breakpoints, $width) {
      $width: map_get($breakpoints, $width);
      @if $type == max {
        $width: $width - 1px;
      }
      @media only screen and (#{$type}-width: $width) {
        @content;
      }
    }
  }

.my_elem{
color:blue;
@include mq(mobile){
    color:red;
 }
}
@viktooar  check ,this light version media-query

}

File: package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "portfolio",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (fabysshk) [inline package.json]: m remove this

File: postcss.config.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  plugins: [
+    require('autoprefixer')({
+      grid: true
```
Comment (fabysshk) [inline postcss.config.js]: m... ie11 support?

File: src/index.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
```
Comment (fabysshk) [inline src/index.html]: <meta name="viewport"
          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">

**Comment by DZakh** [comment]
Don't pull into master

File: postcss.config.js (hunk 3 above)
Comment (DZakh) [reply postcss.config.js]: grid: true? 
I don't know whether it helps, I didn't check

**Pull request closed without merging**